datafusion/core/tests/sql/information_schema.rs

```diff
@@ -436,15 +444,40 @@ async fn information_schema_show_table_table_names() {
     );
 }
 
+//#[tokio::test]
```

Comment:
did you mean to leave this commented out?

Reply:
yes, i thought we might have a way to check whether a variable exists or not in the future